scripts/update-po.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# -------------------------------------------------------------------------------------- #
+#         ___            _     ___                                                       #
+#         |   |   \/    | ) |  |             This software may be modified and distri-   #
+#     O-  |-  |   |  -  |   |  |-  -O        buted under the terms of the MIT license.   #
+#         |   |_  |     |   |  |_            See the LICENSE file for details.           #
+#                                                                                        #
+# -------------------------------------------------------------------------------------- #
+
+# This script takes 'po/flypie.pot' and compiles the latest '.po' file from it.
+# Usage: update-po.sh <LANG-CODE>, where <LANG-CODE> is the language code of the file
+# you want to update. Pass 'all' to update all '.po' files.
```

Comment:
```suggestion
# Usage: update-po.sh -l <LANG-CODE>, where <LANG-CODE> is the language code of the file
# you want to update. Pass '-a' to update all '.po' files.
```

Reply:
Yeah, I had another way of doing it first, this is a relic of that xD